.changeset/purple-kangaroos-shake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@vue-storefront/nuxt": major
+"@vue-storefront/sdk": minor
+---
+
+cdnCacheBustingId option of sdk is now optional, updated nuxt to the latest sdk version
```

Comment:
```suggestion
[CHANGED]: cdnCacheBustingId option of sdk is now optional
[CHANGED]: updated nuxt to the latest sdk version
```

Reply:
Sorry I've forgot to push a fix for that